module/data/item/templates/physical-item.mjs

```diff
@@ -38,12 +38,25 @@ export default class PhysicalItemTemplate extends SystemDataModel {
             {
                 label: "ED.Item.General.price"
             } ),
-            weight: new foundry.data.fields.NumberField( {
-                required: true,
-                nullable: false,
-                min: 0,
-                initial: 0,
-                label: "ED.Item.General.weight"
+            weight: new foundry.data.fields.SchemaField( {
+                value: new foundry.data.fields.NumberField( {
+                    required: true,
+                    nullable: false,
+                    min: 0,
+                    initial: 0,
+                    label: "ED.Item.General.weight"
+                } ),
+                weightMultiplier: new foundry.data.fields.NumberField( {
+                    required: true,
+                    nullable: false,
+                    min: 0,
+                    initial: 0,
```

Comment:
`min`  and `initial` should be 1, since that's the identity element for multiplication

Reply:
correct